### DISCUSSION TITLE
Create presentation?

### DISCUSSION THREAD

**Discussion by PrzemyslawKlys** [ShapeCrawler/ShapeCrawler#171] [Q&A]
In all your examples you're showing how to add something to an existing presentation. Is there a way one could do a presentation from scratch? Could you show me how?

**Comment by AdamShakhabov** [comment]
No, creating a presentation from scratch [https://github.com/ShapeCrawler/ShapeCrawler/issues/53](https://github.com/ShapeCrawler/ShapeCrawler/issues/53) is not available yet.

**Comment by PrzemyslawKlys** [comment]
Thank you.
